modules/@apostrophecms/login/index.js

```diff
@@ -678,6 +679,7 @@ module.exports = {
 
       // Awaitable wrapper for req.login. An implementation detail of the login route
       async passportLogin(req, user) {
+        req.res.cookie(`${self.apos.shortName}.loggedIn`, 'true');
```

Comment:
Unfortunately passport modules don't always call this. So to be on the safe side just set it on every request if it is not already set and the user is logged in.

Reply:
That means putting it into one of our middleware functions.

Reply:
Ok I'll change that 👍